src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,156 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_OUT_OF_BOUND_EXCEPTION = "Index: %d out of bound!! Size: %d";
+    private static final int ONE = 1;
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> previous;
+
+        public Node(T item) {
```

Comment:
[When creating a new Node, pass all the fields to the Node constructor.](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#when-creating-a-new-node-pass-all-the-fields-to-the-node-constructor)
Also i think we can make it more closed.